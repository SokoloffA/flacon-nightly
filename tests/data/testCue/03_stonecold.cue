FILE "stonecold.flac" FLAC
PERFORMER "Various"
TITLE "Tribute To Queen - Stone Cold Queen"
  TRACK 01 AUDIO
    TITLE "Robin Zander - Stone Cold Crazy"
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Mickey Thomas - Play The Game"
    INDEX 01 03:38:30
  TRACK 03 AUDIO
    TITLE "Joe Lynn Turner - Fat Bottomed Girls"
    INDEX 01 08:35:35
  TRACK 04 AUDIO
    TITLE "Geoff Tate - Somebody To Love"
    INDEX 01 13:02:10
  TRACK 05 AUDIO
    TITLE "Gunnar & Mathew Nelson - Crazy Little Thing Called Love"
    INDEX 01 17:47:17
  TRACK 06 AUDIO
    TITLE "Jack Blades - Fight From Inside"
    INDEX 01 20:55:62
  TRACK 07 AUDIO
    TITLE "Jason Scheff - You're My Best Friend"
    INDEX 01 24:48:12
  TRACK 08 AUDIO
    TITLE "Kip Winger - I'm In Love With My Car"
    INDEX 01 27:57:45
  TRACK 09 AUDIO
    TITLE "Glenn Hughes - Killer Queen"
    INDEX 01 31:42:05
  TRACK 10 AUDIO
    TITLE "Tommy Shaw - Spread Your Wings"
    INDEX 01 35:12:15
  TRACK 11 AUDIO
    TITLE "Jack Russel - We Will Rock You"
    INDEX 01 39:58:10
