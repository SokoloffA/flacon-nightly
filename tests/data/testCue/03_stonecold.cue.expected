[ DISC 01 / TRACK 01 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 00:00:00
	INDEX CD 01 = 00:00:00
	TITLE       = Stone Cold Crazy
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Robin Zander
	DATE        =
	DISCID      =
	TRACKNUM    = 1
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 02 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 03:38:30
	INDEX CD 01 = 03:38:30
	TITLE       = Play The Game
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Mickey Thomas
	DATE        =
	DISCID      =
	TRACKNUM    = 2
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 03 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 08:35:35
	INDEX CD 01 = 08:35:35
	TITLE       = Fat Bottomed Girls
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Joe Lynn Turner
	DATE        =
	DISCID      =
	TRACKNUM    = 3
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 04 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 13:02:10
	INDEX CD 01 = 13:02:10
	TITLE       = Somebody To Love
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Geoff Tate
	DATE        =
	DISCID      =
	TRACKNUM    = 4
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 05 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 17:47:17
	INDEX CD 01 = 17:47:17
	TITLE       = Crazy Little Thing Called Love
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Gunnar & Mathew Nelson
	DATE        =
	DISCID      =
	TRACKNUM    = 5
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 06 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 20:55:62
	INDEX CD 01 = 20:55:62
	TITLE       = Fight From Inside
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Jack Blades
	DATE        =
	DISCID      =
	TRACKNUM    = 6
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 07 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 24:48:12
	INDEX CD 01 = 24:48:12
	TITLE       = You're My Best Friend
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Jason Scheff
	DATE        =
	DISCID      =
	TRACKNUM    = 7
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 08 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 27:57:45
	INDEX CD 01 = 27:57:45
	TITLE       = I'm In Love With My Car
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Kip Winger
	DATE        =
	DISCID      =
	TRACKNUM    = 8
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 09 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 31:42:05
	INDEX CD 01 = 31:42:05
	TITLE       = Killer Queen
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Glenn Hughes
	DATE        =
	DISCID      =
	TRACKNUM    = 9
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 10 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 35:12:15
	INDEX CD 01 = 35:12:15
	TITLE       = Spread Your Wings
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Tommy Shaw
	DATE        =
	DISCID      =
	TRACKNUM    = 10
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 11 ]
	FILE        = stonecold.flac
	INDEX CD 00 = 39:58:10
	INDEX CD 01 = 39:58:10
	TITLE       = We Will Rock You
	ALBUM       = Tribute To Queen - Stone Cold Queen
	PERFORMER   = Jack Russel
	DATE        =
	DISCID      =
	TRACKNUM    = 11
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1
